model/registry/otel.yaml

```diff
@@ -1,4 +1,28 @@
 groups:
+  - id: registry.otel
+    prefix: otel
+    type: resource
```

Comment:
nit and not blocking: not sure if we're updating type as a part of this move, but I *think* everything in the registry should be an `attribute_group`.

Reply:
good catch, it should be generic attribute_group

Reply:
Fixed!